### PR TITLE
remove OpenMS checkout from contrib

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

File: dockerfiles/pyopenms/manylinux/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM quay.io/pypa/manylinux2014_x86_64
 
-ARG OPENMS_BRANCH="develop"
+#defaults, can be overwritten from outside
+ARG OPENMS_BRANCH="develop"
```
Comment (jpfeuffer) [inline dockerfiles/pyopenms/manylinux/Dockerfile:4]: Master

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: dockerfiles/pyopenms/manylinux/Dockerfile (hunk 1 above)
Comment (timosachsenberg) [inline dockerfiles/pyopenms/manylinux/Dockerfile:4]:
```suggestion
ARG OPENMS_BRANCH="master" 
```

Comment (jpfeuffer) [reply dockerfiles/pyopenms/manylinux/Dockerfile]: Please change this in the actions too.

**Comment by jpfeuffer** [comment]
release branch logic is missing: https://github.com/OpenMS/contrib/pull/128/files#diff-7b992417dcf3f0b87a2010ce49524835584e898ce03cfbd7b696405925441a4cR42

Now could be a good time to add it. Or use the checked-out sources, then you don't even need a branching logic.

**Comment by timosachsenberg** [comment]
what additional logic is requires? the upper parts seems to work with release tags.
need more information how that logic should look like :)

https://github.com/OpenMS/contrib/pull/128/files#diff-7f20fd9f07cc5015000c27a19f74f5eaf2cd60e776bf9449dd6014fbd5a6a9b5R42

**Comment by jpfeuffer** [comment]
there are no arguments passed to docker. it will always use master.

**Pull request merged**